Flagsmith.Client.Test/FlagsmithTest.cs

```diff
@@ -0,0 +1,186 @@
+﻿using Moq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Threading;
+using Xunit;
+namespace Flagsmith.FlagsmithClientTest
+{
+    public class FlagsmithTest
+    {
+        [Fact]
+        public void TestFlagsmithStartsPollingManagerOnInitIfEnabled()
+        {
+            var mockHttpClient = HttpMocker.MockHttpResponse(new HttpResponseMessage
+            {
+                StatusCode = System.Net.HttpStatusCode.OK,
+                Content = new StringContent(Fixtures.JsonObject.ToString())
+            });
+            var flagsmithClientTest = new FlagsmithClient(Fixtures.ApiKey, httpClient: mockHttpClient.Object, enableClientSideEvaluation: true);
+            mockHttpClient.Verify(x => x.SendAsync(It.IsAny<HttpRequestMessage>(), It.IsAny<CancellationToken>()), Times.Once);
```

Comment:
Can we use a mocking library here to mock the polling manager instead of checking that an HTTP request is made? 

If not, can we at least check which endpoint is being hit here? It's a bit confusing here as the name of the test is talking about starting the polling manager but then all we're checking is that an HTTP request is made 😕